.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: build
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-18.04
```

Comment:
Why an old version of Ubunutu? Is this the most up to date version that GH action supports?

Reply:
according to https://github.com/actions/virtual-environments
we can update to Ubuntu 20.04

Reply:
Cool, let's do it.

Reply:
on it